var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -50,6 +50,10 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
+    version('3.3',
+            'b17bd452593aada93dc0fee748fcfbbf4f04ce3e7d77fdd0341cc9103bcacd0b',
+            url='http://goo.gl/Vrpsns', extension='.tgz')
```

Comment:
I am not sure I can. In fact, I am not sure why the old .tar.gz instances even work. I can't confirm there are files of that name at the specified URL. I guess I could try to wget or curl a file with .tar.gz extension and if it succeeds, assume its there. I *can* confirm there is a file with a .tgz extension. I assumed maybe the package developers changed their practice since last versions. Does it really matter?